packages/storage-events/src/state-to-commands.js

```diff
@@ -0,0 +1,21 @@
+module.exports = stateToCommands;
```

Comment:
ah.. state-to-commands = `command -> event`? mmhhh, maybe `command-processors` or something like that?

Reply:
Actually it is taking a state / list of todos and generates the necessary commands to create a tree out of it...?